monitor/scripts/getBridgeStats.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+cd $(dirname $0)/..
+
+if /usr/local/bin/docker-compose ps | grep -q -i 'monitor'; then
+  for file in configs/*.env
+  do
+    docker run --rm --env-file $file -v $(pwd)/responses:/mono/monitor/responses monitor_monitor /bin/bash -c 'yarn check-all'
```

Comment:
Did you deploy and run this file on a real (not e2e) environment? is it ok that we don't allocate a tty with the `-t` option? Previously it was used with `docker-compose`.

Reply:
Yes, I deployed and run this file on a real environment. By default `docker-compose exec` allocates a TTY, the parameter `-T` [disables this allocation](https://docs.docker.com/v17.09/compose/reference/exec/). In the case of `docker run` it is disabled by default and [it can be enabled using `-t` parameter](https://docs.docker.com/v17.12/edge/engine/reference/commandline/run/#examples). So by not providing this parameter, the behaviour is the same as before.

Reply:
ok. Thanks for information.